code-submissions-clustering-core/src/test/kotlin/org/jetbrains/research/code/submissions/clustering/cluster/ClusteringTest.kt

```diff
@@ -194,5 +215,22 @@ class ClusteringTest : ParametrizedBaseWithUnifierTest(getTmpProjectDir()) {
                 ))
             ),
         )
+
+        @JvmStatic
+        fun getTestDataFromBin(): List<Arguments> {
+            val binDir = "${Paths.get("").toAbsolutePath()}/src/test/resources/bin.data"
```

Comment:
Please, use the Java-embedded [functionality](https://stackoverflow.com/questions/15749192/how-do-i-load-a-file-from-resource-folder) to get the resources folder